### PR TITLE
Release 1.1.0

### PR DESCRIPTION
**Pull request by bastelfreak**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -4,7 +4,24 @@ All notable changes to this project will be documented in this file.
...
+
+[Full Changelog](https://github.com/voxpupuli/puppet-puppet_certificate/compare/v1.0.0...v1.1.0)
+
+**Implemented enhancements:**
+
+- Add Puppet 8 support [\#17](https://github.com/voxpupuli/puppet-puppet_certificate/pull/17) ([bastelfreak](https://github.com/bastelfreak))
+
+**Closed issues:**
+
+- New release to the forge [\#6](https://github.com/voxpupuli/puppet-puppet_certificate/issues/6)
```
Comment (evgeni) [inline CHANGELOG.md:17]: skip?

File: CHANGELOG.md [hunk 2]
```diff
@@ -4,7 +4,24 @@ All notable changes to this project will be documented in this file.
...
+
+**Merged pull requests:**
+
+- README.md: Update code example [\#23](https://github.com/voxpupuli/puppet-puppet_certificate/pull/23) ([bastelfreak](https://github.com/bastelfreak))
+- Document PE workaround for auth.conf [\#19](https://github.com/voxpupuli/puppet-puppet_certificate/pull/19) ([bastelfreak](https://github.com/bastelfreak))
```
Comment (evgeni) [inline CHANGELOG.md:22]: enhancement? docs? :)

File: CHANGELOG.md (hunk 1 above)
Comment (bastelfreak) [reply CHANGELOG.md:17]: okay

**Review by evgeni — APPROVED** [review]

File: CHANGELOG.md (hunk 2 above)
Comment (bastelfreak) [reply CHANGELOG.md:18]: I added the docs label but we don't handle that in the changelog. those are only documentational changes, not sure if those qualify as an enhancement.

Comment (evgeni) [reply CHANGELOG.md:18]: they kinda are, even if small, but I am fine either way

**Pull request merged**